lib/vagrant/util/is_port_open.rb

```diff
@@ -13,7 +13,7 @@ module IsPortOpen
       # @param [Integer] port Port to check.
       # @return [Boolean] `true` if the port is open (listening), `false`
       #   otherwise.
-      def is_port_open?(host, port)
+      def self.is_port_open?(host, port)
```

Comment:
With this being an already existing module, this would be a breaking change. Instead, just add an `extend` to the end:

```ruby
module Vagrant
  module Util
    module IsPortOpen
      def is_port_open?
      ...
      end
      extend IsPortOpen
    end
  end
end
```